stories/components/SInputText.01_Playground.story.vue

```diff
@@ -164,12 +169,16 @@ function initState() {
           :placeholder="state.placeholder"
           :unit-before="state.unitBefore"
           :unit-after="state.unitAfter"
+          :align="state.align"
+          :disabled="state.disabled"
```

Comment:
Took me quite a while to realize that these other props aren't added to the component, despite the controls being there… Though it was a bug or something 😅
Need to check if other playgrounds have the same issue too.
But for now, I think it suffices to only add the missing ones when we encounter any during development.

Reply:
Ah, thanks!